tests/testthat/test_table_aet04.R

```diff
@@ -0,0 +1,261 @@
+# # Data pre-processing
+# adsl <- adsl_raw
+# adae <- adae_raw %>%
+#   dplyr::mutate(
+#     AEDECOD = as.character(AEDECOD),
+#     AEBODSYS = as.character(AEBODSYS),
+#   )
+# gr_grp <- list(
+#   "- Any Grade -" = c("1", "2", "3", "4", "5"),
+#   "Grade 1-2" = c("1", "2"),
+#   "Grade 3-4" = c("3", "4"),
+#   "Grade 5" = "5"
+# )
+#
+# # Raw result for future pruning
+# raw_result <- basic_table() %>%
+#   split_cols_by("ACTARM") %>%
+#   add_colcounts() %>%
+#   count_occurrences_by_grade(
+#     var = "AETOXGR",
+#     grade_groups = gr_grp
+#   ) %>%
+#   split_rows_by("AEBODSYS",
+#                 split_fun = trim_levels_in_group("AETOXGR"),
+#                 child_labels = "visible", nested = TRUE
+#   ) %>%
+#   summarize_occurrences_by_grade(
+#     var = "AETOXGR",
+#     grade_groups = gr_grp
+#   ) %>%
+#   split_rows_by("AEDECOD",
+#                 split_fun = trim_levels_in_group("AETOXGR"),
+#                 child_labels = "visible", nested = TRUE
+#   ) %>%
+#   summarize_num_patients(
+#     var = "USUBJID",
+#     .stats = "unique",
+#     .labels = "- Any Grade -"
+#   ) %>%
+#   count_occurrences_by_grade(
+#     var = "AETOXGR",
+#     grade_groups = gr_grp[-1],
+#     .indent_mods = -1L
+#   ) %>%
+#   build_table(adae, alt_counts_df = adsl) %>%
+#   sort_at_path(
+#     path = "AEBODSYS",
+#     scorefun = cont_n_allcols,
+#     decreasing = TRUE
+#   ) %>%
+#   sort_at_path(
+#     path = c("AEBODSYS", "*", "AEDECOD"),
+#     scorefun = cont_n_allcols,
+#     decreasing = TRUE
+#   )
+
+### SETUP
+load(paste0(main_pkg_dir, "inst/extdata/aet04.Rdata"))
```

Comment:
please safeguard it when file is not present (assuming we don't want to commit it)
Then we need to also skip the test or something

Reply:
so like having a flag that skips all tests if not present?

Reply:
yes something like that

Reply:
Done ;)